src/index.html

```diff
@@ -7,12 +7,57 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to an SCSS file (`index.scss`). SCSS files need to be compiled to CSS before they can be used in an HTML document. Ensure that the SCSS is compiled to a CSS file and link to that CSS file instead.